File: Demo4DotNetCore/Demo4DotNetCore.ResourceServer/Areas/Identity/RequestModel/ClientRequestModel.cs
using Demo4DotNetCore.ResourceServer.Identity.Model;
using Demo4DotNetCore.ResourceServer.Model;

namespace Demo4DotNetCore.ResourceServer.Identity.RequestModel
{
    public class ClientRequestModel : PaginatedRequestModel
    {
        public bool? Enabled { get; set; }
        public string ClientId { get; set; }
        public string ClientName { get; set; }
        public Client Client { get; set; }
    }

}

File: Demo4Angular/src/app/identity/model/client-request.model.ts
import { RequestModel, PaginatedRequestModel, } from 'src/app/shared/request.model';
import { ApiResource, ApiScope, ApiSecret } from './api-resource.model';
import { Client } from './client.model';

export class ClientRequestModel extends PaginatedRequestModel {
    client: Client;
    enabled?: boolean;
    clientId?: string;
    clientName?: string;

    constructor(url: string, pageIndex?: number, pageSize?: number, orderBy?: string, direction?: string) {
        super(url, pageIndex, pageSize, orderBy, direction);
    }
}

